### ISSUE TITLE
Give me a quick way to see the price of a card

### ISSUE THREAD

**Issue by alexanelli** [accassid/turnbased-alpha#4]
For example,  If I'm looking at my deck my instinct is to hover over the card and click on it expecting to see some sort of "more info" type modal pop out that has pricing (tcgplayer mid, etc) on it

**Comment by alexanelli** [comment]
You can put buy links here too

**Comment by alexanelli** [comment]
Just figured out that double clicking opens card info, that's what I was looking for.  Not a fan of the double click though I didn't figure that out until I saw that there was a card stats on the right popout when looking at deck stats, so I just spammed cards until I figured out how.  I'd like to see maybe the side window open on a single click, but a single click anywhere else on page closes it (unless you click on another card obviously).  Also definitely still would love pricing but I understand thats more a matter of getting the tcgplayer api keys once names and etc are settled.   just my .02
